modules/net-cloudnat/variables.tf

```diff
@@ -102,3 +108,15 @@ variable "subnetworks" {
   }))
   default = []
 }
+
+variable "enable_endpoint_independent_mapping" {
+  description = "Enables endpoint independent mapping if null"
+  type        = bool
+  default     = null
+}
+
+variable "enable_dynamic_port_allocation" {
+  description = "Disables dynamic port allocation if null"
```

Comment:
variable name says "enable", description days "disable". Which one is it?

Reply:
If `null` it's disabled, but if `true` it's enabled -- this is a simple passthrough of the Terraform resource [here](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_router_nat#enable_dynamic_port_allocation), and mirrors its behavior (enable_endpoint_independent_mapping defaults to true, and the two are mutually exclusive, meaning that enable_dynamic_port_allocation defaults to false).  

This is a non-issue in the updated PR with a config block.